src/lib/protocols/credentials/__tests__/StubWallet.ts

```diff
@@ -4,6 +4,12 @@ import { UnpackedMessageContext } from '../../../types';
 
 export class StubWallet implements Wallet {
   private wh?: number | undefined;
```

Comment:
Maybe we can also rename it to `_walletHandle` to keep consistency with other getters. But I don't have a strong opinion here 🤷‍♂️

Reply:
Updated